packages/world/src/utils/ForceFieldUtils.sol

```diff
@@ -104,6 +104,7 @@ library ForceFieldUtils {
    */
   function destroyForceField(EntityId forceField) internal {
```

Comment:
Is this getting used anywhere?

Reply:
it should 🫠 I'm replacing it with direct calls to the tables, as it should apply to all machines